TM1py/Services/CellService.py

```diff
@@ -191,50 +195,47 @@ def get_value(self, cube_name: str, element_string: str, dimensions: List[str] =
         :return:
         """
         mdx_template = "SELECT {} ON ROWS, {} ON COLUMNS FROM [{}]"
-        mdx_rows_list = []
-        if not dimensions:
-            dimensions = self.get_dimension_names_for_writing(cube_name=cube_name)
-        element_selections = element_string.split(element_separator)
-
-        # Build the ON ROWS statement:
-        # Loop through the comma separated element selection, except for the last one
-        for dimension_name, element_selection in zip(dimensions[:-1], element_selections[:-1]):
-            if hierarchy_separator not in element_selection:
-                if hierarchy_element_separator in element_selection:
-                    hierarchy_name, element_name = element_selection.split(hierarchy_element_separator)
-                else:
-                    hierarchy_name = dimension_name
-                    element_name = element_selection
+        mdx_strings_list = []
 
-                mdx_rows_list.append("{[" + dimension_name + "].[" + hierarchy_name + "].[" + element_name + "]}")
+        # Keep backwards compatibility with the earlier used "element_string" parameter
+        if elements is None and "element_string" in kwargs:
+            elements = kwargs.get("element_string")
```

Comment:
Just one small change request: Please use `kwargs.pop` here so that the element isn't delegated further down.

Reply:
Thanks for the input, Marius!
I changed it to ```kwargs.pop```.

Did some tiny changes on the comments while I had the chance 😄

Reply:
Nice work. Thanks!